terracumber/terraformer.py

```diff
@@ -85,8 +85,10 @@ def inject_repos(self, custom_repositories_json):
                         (new_line, n) = subn(placeholder, replacement, line)
                         print(new_line, end='')
                         n_replaced += n
-                    if n_replaced != 1:
+                    if n_replaced > 1:
                         return 2
+                    elif n_replaced == 0:
+                        return 3
```

Comment:
The function docstring state that return codes are 0 , 1  and 2.
I don't know if returning 3 here might be a problem or we don't really care (eg: we only check if the return is 0 or not when we call this function).

Just pointing it out :)

Reply:
Nevermind, I see you added a check for when the return is 3.
All that is missing then is just mentioning it in the doctstring.

Reply:
In the future, it would be nicer to have proper Error classes raised at that point. Make sense?